info-compiler/src/main/java/com/google/sps/data/NewsArticle.java

```diff
@@ -17,7 +17,7 @@
 import java.util.Arrays;
 import java.util.List;
 
-/** 
```

Comment:
lol are all these linting stuff?

Reply:
These are actually before linters take effect lol. But I just suddenly noticed that there is an extra space after each javadoc comment block